src/styles.scss

```diff
@@ -1,10 +1,16 @@
 $fa-font-path : '../node_modules/font-awesome/fonts';
 @import '../node_modules/font-awesome/scss/font-awesome';
+@import './app/lib/colors.scss';
+@import './app/lib/tables.scss';
+
 
 html,
 body {
     font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-    margin: 0;    
+    margin: 0;
+    height: 100%;
```

Comment:
this height 100% causes there to always be a scroll bar 

![image](https://user-images.githubusercontent.com/1934453/35642481-7b047fa4-0680-11e8-8eaf-70d7fd015888.png)